### PR TITLE
feat: Add HP3325A Function Generator

### PR DESCRIPTION
**Pull request by fundthmcalculus**
* Adds support for all functions on the HP3325A (I don't have an HP3325B to test against)
* Verified functionality against my HP3325A
* Would love opinions on whether to use properties as shown here, or attempt to use `bool_property` `enum_property` similar to the HP3456A implementation.
* Fixed some spelling / grammar issues in various files that I came across.

### CODE REVIEW

File: src/instruments/util_fns.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def bool_property(
     """
 
     def _getter(self):
-        return self.query(command + "?").strip() == inst_true
+        return self.query(command).strip() == inst_true
```
Comment (fundthmcalculus) [inline src/instruments/util_fns.py:211]:
```suggestion
        return self.query(command + "?").strip() == inst_true
```

**Comment by fundthmcalculus** [comment]
@scasagrande sorry for the, uh, year-long delay, but the package is updated as you requested. I'll be adding a few other older HP instruments I have in my home lab in the near future, since I have more free time (in between jobs).

**Comment by scasagrande** [comment]
no worries! I hope everything is okay 😄

**Review by scasagrande — APPROVED** [review]
i gave it a quick look and it seems fine to me!

**Comment by scasagrande** [comment]
looks like just a bit missing from your test coverage https://app.codecov.io/gh/instrumentkit/InstrumentKit/pull/392/blob/src/instruments/hp/hp3325a.py

**Comment by fundthmcalculus** [comment]
> looks like just a bit missing from your test coverage https://app.codecov.io/gh/instrumentkit/InstrumentKit/pull/392/blob/src/instruments/hp/hp3325a.py

Thanks for the clarification, fixed! 💪 Now that I know how the layout is supposed to work, I'll be adding a few more HP/Agilent instruments.

**Comment by scasagrande** [comment]
fantastic! thank you

**Pull request merged**